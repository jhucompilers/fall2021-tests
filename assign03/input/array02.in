PROGRAM array02;

    TYPE Row = RECORD arr: ARRAY 3 OF CHAR; END;
    VAR board: ARRAY 3 OF Row;

    VAR i, j : INTEGER;

BEGIN
    i := 0;
    WHILE i <= 3 DO
        j := 0;
        WHILE j <= 3 DO
            READ board[i].arr[j];
            j := j + 1;
        END;
        i := i + 1;
    END;
END.