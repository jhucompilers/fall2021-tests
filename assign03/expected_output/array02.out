1,VAR,arr,ARRAY 3 OF CHAR
0,TYPE,Row,RECORD (ARRAY 3 OF CHAR)
0,VAR,board,ARRAY 3 OF RECORD (ARRAY 3 OF CHAR)
0,VAR,i,INTEGER
0,VAR,j,INTEGER
